src/mod.ts

```diff
@@ -1,120 +1,133 @@
-import { Vec3, ColorType,Vec2, Geometry, activeDiffGet, Text, ModelScene, GeometryMaterial, AnimatedObjectInput} from "https://deno.land/x/remapper@3.1.2/src/mod.ts"
+import { activeDiffGet, AnimatedObjectInput, ColorType, Geometry, GeometryMaterial, KeyframesVec3, ModelScene, RawGeometryMaterial, Text, Vec2 } from "https://deno.land/x/remapper@3.1.2/src/mod.ts";

```

Comment:
Imports were missing `KeyframesVec3` and `RawGeometryMaterial`, also the formatted just put everything in alphabetical order.